Common/Data/Custom/Intrinio/IntrinioConfig.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuantConnect.Data.Custom.Intrinio
+{
+    /// <summary>
+    ///     Auxiliary class to access all Intrinio API data.
+    /// </summary>
+    public static class IntrinioConfig
+    {
+        /// <summary>
+        ///     Intrinio API user
+        /// </summary>
+        public static string User { get; set; }
+        /// <summary>
+        ///     Intrinio API password
+        /// </summary>
+        public static string Password { get; set; }
+
+        /// <summary>
+        ///     Check if Intrinio API user and password are not empty or null.
+        /// </summary>
+        public static bool AreUserAndPasswordSet => !(string.IsNullOrWhiteSpace(User) && string.IsNullOrWhiteSpace(Password));
```

Comment:
not required, just me being nit picky :)
Naming -> just thinking of the if statement:
```csharp
if (IntrinioConfig.AreUserAndPasswordSet) { }
```
Or even
```csharp
if (IntrinioConfig.IsInitialized) { }
```

Might read better without the `Are` prefix.


Also, should this be `!(string.IsNullOrWhiteSpace(User) || string.IsNullOrWhiteSpace(Password)` ?

Consider user is null and password is set, we would want this to return `false`, but I think it would return true:
`! ( true && false )` => `!(false)` -> `true`

VS
`! (true || false )` => !(true)` -> `false` as desired

Another way to write it would be to lose the parenthesis:
```csharp
public static bool IsInitialized => !string.IsNullOrWhiteSpace(User) && !string.IsNullOrWhiteSpace(Password)
```